cfme/markers/env_markers/provider.py

```diff
@@ -36,6 +36,7 @@ def _param_check(metafunc, argnames, argvalues):
 
     """
     # If no parametrized args were named, don't parametrize
+    assert isinstance(argvalues, list)
```

Comment:
**REQUIRED:** This is going to blow up with an ```AssertionError``` what causes that and why was it OK to not blow up before? Or what was the bad consequence of not blowing up?